build/NuSpecs/Microsoft.ProjectReunion.Foundation.props

```diff
@@ -2,5 +2,5 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- <Import Project="$(MSBuildThisFileDirectory)\MicrosoftProjectReunionVersion.props"/> -->
   <!-- <Import Project="$(MSBuildThisFileDirectory)\Common.targets"/> -->
-  <Import Project="$(MSBuildThisFileDirectory)\Microsoft.ApplicationModel.Resources.props"/>
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.ProjectReunion.Foundation.FrameworkPackage.props" Condition="Exists('$(MSBuildThisFileDirectory)Microsoft.ProjectReunion.Foundation.FrameworkPackage.props')" />
```

Comment:
>Microsoft.ProjectReunion.Foundation.FrameworkPackage.props [](start = 46, length = 58)

I just did a git pull --rebase for the GitHub repo, but couldn't find the file Microsoft.ProjectReunion.Foundation.FrameworkPackage.props.

Reply:
that's an optional file. It will be added by aggregation pipeline